### PR TITLE
Initial commit. Note that shared utility methods have been moved.

### PR DESCRIPTION
**Pull request by laurejt**

### CODE REVIEW

File: scripts/evaluate_ocr.py [hunk 1]
```diff
@@ -0,0 +1,157 @@
...
+                    vocab.append(t.text.lower())
+        return set(vocab)
+
+    def dict_lookup(self, text, min_chars=3):
+        """
+        Return percentage of (lower-cased, min_char+ length) tokens in
+        reference dictionary; if there are no such tokens, return -1
+        """
+        num = 0
+        den = 0
```
Comment (rlskoeser) [inline scripts/evaluate_ocr.py:51]: is `den` short for something?

File: README.md [hunk 2]
```diff
@@ -18,4 +18,23 @@ pip install -e ".[dev]"
...
+
+For example, to create and activate the `ppa-ocr` environment, run the following commands:
+```
+conda env create -f scripts/envs/ppa-ocr.yml
+conda activate ppa-ocr
+```
+
+### Script descriptions
+- `get_character_stats.py`: Compiles character-level statistics for a PPA text (sub)corpus
+  - Environment: `ppa-ocr`
```
Comment (rlskoeser) [inline README.md:38]:
```suggestion
  - Environment: `ppa-ocr`
  - Download Spacy English language model: `python -m spacy download en_core_web_lg`
```

**Review by rlskoeser — APPROVED** [review]
I was able to follow the directions to create and use a new conda env (worked great once I had a current version of conda!).  I get an error trying to run the script at first because the spacy model wasn't downloaded. Is this something the conda env could handle? 

For one-off scripts seems fine to just document that you need to run the download command first; suggested a change to that effect in the readme.

I was able to start the script; I interrupted it before too long, but I can see the output it generated before I stopped it.

File: scripts/evaluate_ocr.py (hunk 1 above)
Comment (laurejt) [reply scripts/evaluate_ocr.py:51]: num = numerator
den = denominator

**Pull request merged**